addons/survey/models/survey_user_input.py

```diff
@@ -623,6 +625,27 @@ def _message_get_suggested_recipients(self):
                 )
         return recipients
 
+    def _notify_new_participation_subscribers(self):
+        subtype_id = self.env.ref('survey.mt_survey_survey_user_input_completed', raise_if_not_found=False)
+        if not self.ids or not subtype_id:
+            return
+        odoobot_id = self.env.ref('base.partner_root').id
+        # Only post if there are any followers
+        recipients_data = self.env['mail.followers']._get_recipient_data(self.survey_id, 'notification', subtype_id.id)
+        followed_survey_ids = [survey_id for survey_id, followers in recipients_data.items() if followers]
+        for user_input in self.filtered(lambda user_input_: user_input_.survey_id.id in followed_survey_ids):
+            if self.env.user.is_public:
+                author_id = odoobot_id
+                body = _('Someone just participated in "%(survey_title)s".', survey_title=user_input.survey_id.title)
```

Comment:
Note that we could use 'partner_id', 'email' or 'nickname' fields of user_input model to try to be more precise than "someone"